docs/serving/queue-extensions.md

```diff
@@ -0,0 +1,31 @@
+# Queue Proxy Runtime Extensions
+
+Queue Proxy can be built with additional runtime extensions. Such extensions add optional features to Queue Proxy. While Knative core upstream introduce core features available on any knative deployment, Queue Proxy Extensions allow adding additional runtime features. It is up to Downstream to decide which such extensions will be available on what package.
```

Comment:
```suggestion
Queue Proxy can be built with additional runtime extensions enabled that provide optional features.
```
I'd simplify this by a lot and instead explain what Queue Proxy is and why it matters in Knative. The stuff about downstream etc should be explained by different vendors in their downstream docs instead, i.e. they can call out which additional features if any are available in their offering. For core Knative docs we should instead explain the feature and what the possible options are.